File: sdk/typescript/examples/06_kv_counter.mjs
// 06_kv_counter:原子计数器。
import { client } from "./00_helpers.mjs";
const combee = client();
const cell = await combee.cells.create();
console.log("incr:", await cell.kv.incr("pageviews"));
console.log("incr+5:", await cell.kv.incr("pageviews", 5));
console.log("decr-2:", await cell.kv.decr("pageviews", 2));

File: sdk/python/examples/06_kv_counter.py
"""06_kv_counter:原子计数器。"""
from helpers import client

combee = client()
cell = combee.cells.create()
print("incr:", cell.kv.incr("pageviews"))
print("incr+5:", cell.kv.incr("pageviews", 5))
print("decr-2:", cell.kv.decr("pageviews", 2))
